pkg/rehearse/jobs.go

```diff
@@ -353,28 +438,32 @@ func replaceClusterProfiles(volumes []v1.Volume, profiles []config.ConfigMapSour
 type Executor struct {
 	Metrics *ExecutionMetrics
 
-	dryRun     bool
-	rehearsals []*prowconfig.Presubmit
-	prNumber   int
-	prRepo     string
-	refs       *pjapi.Refs
-	loggers    Loggers
-	pjclient   pj.ProwJobInterface
+	dryRun         bool
+	allowPeriodics bool
+	presubmits     []*prowconfig.Presubmit
+	periodics      []prowconfig.Periodic
```

Comment:
Why `[]*` vs. `[]`?

Reply:
The incorrect one is the `[]*prowconfig.Presubmit` which we return [here](https://github.com/openshift/ci-tools/blob/master/pkg/rehearse/jobs.go#L90). Unfortunately,  we did more changes based on that(metrics, executor etc...) and we need another PR to clear this up.